src/internet_identity/internet_identity.did

```diff
@@ -532,6 +521,72 @@ type SignedIdAlias = record {
     id_dapp : principal;
 };
 
+type IdRegNextStepResult = record {
```

Comment:
Why not `IdRegStartResult`?

Reply:
I reuse this result for `_start` and `check_captcha`. It is essentially a result to indicate a step has been completed correctly and what the next step is.

So if there are more / different steps in the flow in the future, they can also use this.